test/code_size/hello_webgl_wasm2js.json

```diff
@@ -1,10 +1,10 @@
 {
   "a.html": 567,
   "a.html.gz": 379,
-  "a.js": 17454,
-  "a.js.gz": 7873,
+  "a.js": 17450,
+  "a.js.gz": 7869,
   "a.mem": 3171,
   "a.mem.gz": 2713,
-  "total": 21192,
-  "total_gz": 10965
+  "total": 21188,
+  "total_gz": 10961
```

Comment:
Are these code size changes actually due to this change?  If not I would not include them.

Reply:
I am not sure. I am just experimenting trying to get the tests to pass.

Reply:
I see:
```
test_minimal_runtime_code_size_math (test_other.other) ... FAIL                    3512 != 3511
test_minimal_runtime_code_size_random_printf_wasm (test_other.other) ... FAIL     12766 != 12774
test_minimal_runtime_code_size_random_printf_wasm2js (test_other.other) ... FAIL  17344 != 17351
```

Reply:
Do you see those failures on main, or only your branch?

Reply:
I think locally they fail in both cases. But the failures I indicated above are CI failures caused by this patch.

Reply:
In order to keep get accurate code size number locally I recommend using `tot` build of emsdk (if you use emsdk).     If you don't use emsdk then instead you will want to make sure you have llvm and binaryen both built from `main`.   You will also want to do `./emcc --clear-cache` before running the test.

I normally run this command: `./test/runner other.*code_size* other.*metadce*  --rebase`.    You the files that are checked in are up-to-date and you have a tot build of llvm and binaryen that command should not generate any changes.

Reply:
Hmm .. now it looks like these change improves code size which looks very odd give that it only adds code. 

Let me download this PR and see if actually affects any of these code size test..

Reply:
I verfied that this change only seems to effect `test_metadce_hello_dylink` and even then well with the tolerance.  

So I think can you safely revert all these expectation files.

Reply:
Okay reverted it. I still don't understand the test failures.